presto-main/src/test/java/com/facebook/presto/operator/scalar/TestStringFunctions.java

```diff
@@ -381,6 +381,54 @@ public void testUpper()
     }
 
     @Test
+    public void testLeftPad()
+    {
+        assertFunction("LPAD('text', 5, 'x')", VARCHAR, "xtext");
+        assertFunction("LPAD('text', 4, 'x')", VARCHAR, "text");
+
+        assertFunction("LPAD('text', 6, 'xy')", VARCHAR, "xytext");
+        assertFunction("LPAD('text', 7, 'xy')", VARCHAR, "xyxtext");
+
+        assertFunction("LPAD('\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ', 10, '\u671B')", VARCHAR, "\u671B\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ");
+        assertFunction("LPAD('\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ', 11, '\u671B')", VARCHAR, "\u671B\u671B\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ");
+        assertFunction("LPAD('\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ', 12, '\u5E0C\u671B')", VARCHAR, "\u5E0C\u671B\u5E0C\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ");
+        assertFunction("LPAD('\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ', 13, '\u5E0C\u671B')", VARCHAR, "\u5E0C\u671B\u5E0C\u671B\u4FE1\u5FF5 \u7231 \u5E0C\u671B  ");
+
+        // truncation
+        assertFunction("LPAD('text', 3, 'xy')", VARCHAR, "tex");
```

Comment:
The common `left` / `right` functions can provide the alternate truncation behavior.